recipe/meta.yaml

```diff
@@ -19,8 +19,9 @@ source:
     - patches/0018-Remove-runtime-agent-dependencies.patch
 
 build:
-  number: 0
-  skip: true  # [py<39 or py>312]
+  number: 1
+  # All Python versions are breaking the CI. Temporarily, only Python 3.13 will be used to rebuild and release these artifacts.
+  skip: true  # [py<313]
```

Comment:
You can use the same build number without changing skip section but add any comment line.
Then only py313 will be built, because all other python variants already exist on the main channel